mobile-app/cypress/integration/functional/wallet/balances/announcements.spec.ts

```diff
@@ -116,4 +117,145 @@ context('Wallet - Balances - Announcements', () => {
     cy.getByTestID('announcements_banner').should('exist')
     cy.getByTestID('announcements_text').should('contain', 'Guidelines')
   })
+
+  it('should display blockchain is down warning message with no existing announcement', () => {
+    cy.intercept('**/announcements', {
+      statusCode: 200,
+      body: []
+    })
+    cy.getByTestID('announcements_banner').should('not.exist')
+    cy.intercept('**/regtest/stats', {
+      statusCode: 404,
+      body: '404 Not Found!',
+      headers: {
+        'x-not-found': 'true'
+      }
+    })
+    cy.wait(5000)
+    cy.getByTestID('announcements_banner').should('exist')
+    cy.getByTestID('announcements_text').should('contain', 'blockchain')
```

Comment:
```suggestion
    cy.getByTestID('announcements_text').should('contain', 'We are currently investigating a syncing issue on the blockchain. View more details on the DeFiChain Status Page.')
```

similarly for the other string check